.github/workflows/ci.yml

```diff
@@ -1,5 +1,5 @@
 name: CI
-on: [push, pull_request]
```

Comment:
Y?

Reply:
There is no need to run the CI on every push
Currently it's running twice for every PR

Reply:
Reverted